osu.Game/Beatmaps/BeatmapSetOnlineInfo.cs

```diff
@@ -70,6 +70,16 @@ public class BeatmapSetOnlineInfo
         /// The availability of this beatmap set.
         /// </summary>
         public BeatmapSetOnlineAvailability Availability { get; set; }
+
+        /// <summary>
+        /// Beatmap set's song genre.
```

Comment:
What do you think about
```suggestion
        /// The song genre of this beatmap set.
```

Reply:
Sure why not